### PR TITLE
proxy-task-9

### PR DESCRIPTION
**Pull request by AnnaShtyka**

### CODE REVIEW

File: src/services/meteorsService.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+const https = require("https");
+
+const getMeteorsDataWithQuery = require("../utils/meteorsMapper");
+
+const httpsAgent = new https.Agent({
+  rejectUnauthorized: false
+});
+
+const START_DATE = "2024-03-22";
+const END_DATE = "2024-03-29";
```
Comment (mdziarkach) [inline src/services/meteorsService.js]: I don't see any place where you use this constants
only in logs

File: src/services/meteorsService.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        params.count,
+        params.wereDangerousMeteors
+      );
+      console.log(
+        `From ${START_DATE} to ${END_DATE} near earth were ${allElements}`
+      );
+
+      return data;
+    })
+    .catch((error) => console.log(error));
```
Comment (mdziarkach) [inline src/services/meteorsService.js]: throw exception please
controller should catch it and handle in next(error)

File: src/utils/meteorsMapper.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
+const mapAllMeteorsData = (allMeteors) => {
+  const meteors = Object.values(allMeteors.near_earth_objects).flat();
+  return meteors;
+};
+
+const getMeteorData = (meteor) => {
```
Comment (mdziarkach) [inline src/utils/meteorsMapper.js]: looks like mapper or DTO

File: src/utils/meteorsMapper.js [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+const getCloseApproachData = (closeApproachData) => {
+  return closeApproachData.map((data) => ({
+    closeApproachDateFull: data.close_approach_date_full,
+    relativeVelocityInKilometersPerSecond:
+      data.relative_velocity.kilometers_per_second
+  }));
+};
+
+const isWereDangerousMeteors = (meteors) => {
```
Comment (mdziarkach) [inline src/utils/meteorsMapper.js]: maybe `hasDangerousMeteor`?

File: src/services/meteorsService.js (hunk 1 above)
Comment (AnnaShtyka) [reply src/services/meteorsService.js]: Updated

File: src/services/meteorsService.js (hunk 2 above)
Comment (AnnaShtyka) [reply src/services/meteorsService.js]: Updated

File: src/utils/meteorsMapper.js (hunk 3 above)
Comment (AnnaShtyka) [reply src/utils/meteorsMapper.js]: Updated

File: src/utils/meteorsMapper.js (hunk 4 above)
Comment (AnnaShtyka) [reply src/utils/meteorsMapper.js]: Updated

**Review by mdziarkach — APPROVED** [review]

**Pull request merged**